spec/models/alchemy/legacy_page_url_spec.rb

```diff
@@ -5,19 +5,27 @@
 describe Alchemy::LegacyPageUrl do
   let(:page) { build_stubbed(:alchemy_page) }
 
-  let(:page_url_with_parameters) do
```

Comment:
I removed the `let` statements as it just adds clutter. It's better readable without them.